akka-docs/src/main/paradox/additional/deploying.md

```diff
@@ -44,3 +44,27 @@ to see what this looks like in practice.
 ### Resource limits
 
 To avoid CFS scheduler limits, it is best not to use `resources.limits.cpu` limits, but use `resources.requests.cpu` configuration instead.
+
+## When Shutdown/Startup Is Required
```

Comment:
I wonder if we should move this to `rolling-updates.md`, or at least cross-link?

Reply:
I agree that it feels like it's more related to rolling-updates. Cases when rolling updates can't be performed.